<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>xorg-x11-drv-siliconmotion.git - xorg-x11-drv-siliconmotion</title>
<meta name='generator' content='cgit v0.9.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/xorg-x11-drv-siliconmotion.git/atom/0001-Remove-miInitializeBackingStore.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='xorg-x11-drv-siliconmotion.git' href='/cgit/xorg-x11-drv-siliconmotion.git/'>xorg-x11-drv-siliconmotion.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>xorg-x11-drv-siliconmotion</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/xorg-x11-drv-siliconmotion.git/'>summary</a><a href='/cgit/xorg-x11-drv-siliconmotion.git/refs/'>refs</a><a href='/cgit/xorg-x11-drv-siliconmotion.git/log/0001-Remove-miInitializeBackingStore.patch'>log</a><a class='active' href='/cgit/xorg-x11-drv-siliconmotion.git/tree/0001-Remove-miInitializeBackingStore.patch'>tree</a><a href='/cgit/xorg-x11-drv-siliconmotion.git/commit/0001-Remove-miInitializeBackingStore.patch'>commit</a><a href='/cgit/xorg-x11-drv-siliconmotion.git/diff/0001-Remove-miInitializeBackingStore.patch'>diff</a><a href='/cgit/xorg-x11-drv-siliconmotion.git/stats/0001-Remove-miInitializeBackingStore.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/xorg-x11-drv-siliconmotion.git/log/0001-Remove-miInitializeBackingStore.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/xorg-x11-drv-siliconmotion.git/tree/'>root</a>/<a href='/cgit/xorg-x11-drv-siliconmotion.git/tree/0001-Remove-miInitializeBackingStore.patch'>0001-Remove-miInitializeBackingStore.patch</a></div><div class='content'>blob: 32e38fcc5365ce23c0b457c7530cd14c0988dd5f (<a href='/cgit/xorg-x11-drv-siliconmotion.git/plain/0001-Remove-miInitializeBackingStore.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a class='no' id='n1' name='n1' href='#n1'>1</a>
<a class='no' id='n2' name='n2' href='#n2'>2</a>
<a class='no' id='n3' name='n3' href='#n3'>3</a>
<a class='no' id='n4' name='n4' href='#n4'>4</a>
<a class='no' id='n5' name='n5' href='#n5'>5</a>
<a class='no' id='n6' name='n6' href='#n6'>6</a>
<a class='no' id='n7' name='n7' href='#n7'>7</a>
<a class='no' id='n8' name='n8' href='#n8'>8</a>
<a class='no' id='n9' name='n9' href='#n9'>9</a>
<a class='no' id='n10' name='n10' href='#n10'>10</a>
<a class='no' id='n11' name='n11' href='#n11'>11</a>
<a class='no' id='n12' name='n12' href='#n12'>12</a>
<a class='no' id='n13' name='n13' href='#n13'>13</a>
<a class='no' id='n14' name='n14' href='#n14'>14</a>
<a class='no' id='n15' name='n15' href='#n15'>15</a>
<a class='no' id='n16' name='n16' href='#n16'>16</a>
<a class='no' id='n17' name='n17' href='#n17'>17</a>
<a class='no' id='n18' name='n18' href='#n18'>18</a>
<a class='no' id='n19' name='n19' href='#n19'>19</a>
<a class='no' id='n20' name='n20' href='#n20'>20</a>
<a class='no' id='n21' name='n21' href='#n21'>21</a>
<a class='no' id='n22' name='n22' href='#n22'>22</a>
<a class='no' id='n23' name='n23' href='#n23'>23</a>
<a class='no' id='n24' name='n24' href='#n24'>24</a>
<a class='no' id='n25' name='n25' href='#n25'>25</a>
<a class='no' id='n26' name='n26' href='#n26'>26</a>
</pre></td>
<td class='lines'><pre><code>From f19d7e463c30f1364e82e8c9f87b8a8407d53680 Mon Sep 17 00:00:00 2001
From: Adam Jackson &lt;ajax@redhat.com&gt;
Date: Wed, 9 Jan 2013 22:59:39 -0500
Subject: [PATCH] Remove miInitializeBackingStore()

Signed-off-by: Adam Jackson &lt;ajax@redhat.com&gt;
---
 src/smi_driver.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/smi_driver.c b/src/smi_driver.c
index 4794571..134db79 100644
--- a/src/smi_driver.c
+++ b/src/smi_driver.c
@@ -1750,8 +1750,6 @@ SMI_ScreenInit(SCREEN_INIT_ARGS_DECL)
 		   "Done writing mode.  Register dump:\n");
     SMI_PrintRegs(pScrn);
 
-    miInitializeBackingStore(pScreen);
-
 #ifdef HAVE_XMODES
     xf86DiDGAInit(pScreen, (unsigned long)(pSmi-&gt;FBBase + pScrn-&gt;fbOffset));
 #endif
-- 
1.8.0.2

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.9.2 at 2013-12-06 22:38:43 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
